### DISCUSSION TITLE
Allow custom font-face for subtitles

### DISCUSSION THREAD

**Discussion by cacfd3a** [vimeo/player.js#670] [Ideas]
Chromium browsers support cuistom fontfaces in WebVTT. It would be great if Vimeo allows users to customize the typeface of the subtitles.

https://vimeo.zendesk.com/hc/en-us/articles/224968828-Captions-and-subtitles:
> We use Helvetica Neue with bold styling for captioning and subtitle display in the Vimeo player. The size of the font is responsive, meaning it will adjust per the size of your browser window.

**Comment by luwes** [comment]
thanks for the suggestion @digitalheir. we will look into this but it'll probably take some time as it's a very specific feature request. keeping the issue open to gain popularity.

Related issue #267
